wpilib/wpilib/sendablechooser.py

```diff
@@ -1,23 +1,23 @@
-# validated: 2017-10-07 EN 34c18ef00062 edu/wpi/first/wpilibj/smartdashboard/SendableChooser.java
+# validated: 2017-12-16 EN f9bece2ffbf7 edu/wpi/first/wpilibj/smartdashboard/SendableChooser.java
 #----------------------------------------------------------------------------
 # Copyright (c) FIRST 2008-2017. All Rights Reserved.
 # Open Source Software - may be modified and shared by FRC teams. The code
 # must be accompanied by the FIRST BSD license file in the root directory of
 # the project.
 #----------------------------------------------------------------------------
 
-from .sendable import Sendable
+from .sendablebase import SendableBase
 
 __all__ = ["SendableChooser"]
 
-class SendableChooser(Sendable):
+class SendableChooser(SendableBase):
     """A useful tool for presenting a selection of options to be displayed on
-    the SmartDashboard
+    the :class:`.SmartDashboard`
 
     For instance, you may wish to be able to select between multiple
-    autonomous modes. You can do this by putting every possible Command
+    autonomous modes. You can do this by putting every possible :class:`wpilib.command.Command`
```

Comment:
link doesn't work on this one. I tried `..command.Command` also

Reply:
Does `.Command` not work?